colossalai/inference/core/engine.py

```diff
@@ -223,6 +224,9 @@ def add_request(
 
         block_size = self.inference_config.block_size
 
+        if prompts is not None and not isinstance(prompts, list):
```

Comment:
request_id also needs to revise and I suggest using ` if isinstance(prompts,str) and isinstance(requests_id,int):`

Reply:
Done